src/runtime/entitlements-manager.js

```diff
@@ -230,22 +245,29 @@ export class EntitlementsManager {
       default:
         return;
     }
-
-    this.postEntitlementsRequest_(new EntitlementJwt(), result, source);
+    const params = parseQueryString(this.win_.location.search);
+    this.postEntitlementsRequest_(
+      new EntitlementJwt(),
+      result,
+      source,
+      params['gaa_n']
+    );
   }
 
   // Informs the Entitlements server about the entitlement used
   // to unlock the page.
   postEntitlementsRequest_(
     usedEntitlement,
     entitlementResult,
-    entitlementSource
+    entitlementSource,
+    opt_nonce = ''
```

Comment:
Could you rename this to `nonce` or `optionalNonce`? https://google.github.io/styleguide/jsguide.html#naming-parameter-names

Reply:
optionalNonce it is.

Reply:
per Dan `nonce` is officially deprecated it's now `gaaToken` or in this case `optionalGaaToken`